Automattic/jetpack
# Boost: Include dependency

## Proposed changes:

* Fix a conflict with WP Optimize because of a missing dependency

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
- Install both Boost and WP Optimize
- Enable minify CSS on WP Optimize
- Enable minify JS on Boost
- Make sure they both work
